File: tutorial/discover/hello_world.cpp
#include "imgui.h"
#include <hello_imgui/hello_imgui.h>

void Gui() {                                                                // 1.
    ImGui::Text("Hello, World");                                            // 2.
}

int main() {
    HelloImGui::Run(                                                        // 3.
        Gui,             // guiFunction
        "Hello, World!", // windowTitle
        true);           // windowSizeAuto
}

File: pyodide_projects/projects/imgui_bundle_playground/examples/hello_world.py
"""Hello World example:
Create an App in 3 lines of code,
using Dear ImGui and Hello ImGui!
"""
from imgui_bundle import hello_imgui, imgui

def gui():
    imgui.text("Hello, world!")


# The window title will become the HTML title of the rendered page.
hello_imgui.run(gui, window_title="Hello HelloImGui!")
